ui-tests/src/test/resources/features/integrations/integration-import-export-open-api.feature

```diff
@@ -0,0 +1,85 @@
+Feature: API Provider Integration - Import Export
```

Comment:
Please add proper tags such as `@ui,@api-provider` and the maintainer comment (see in other feature files)